tabular/nyc_tlc/download_files.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+wget https://www.dropbox.com/s/80lk5l2ycsu432w/eval.csv?dl=1 -O $1/eval.csv
```

Comment:
If the user forgets to add an output file name, this attempts to download to the root filesystem. 

let's at least check to ensure $1 is not null. Could also use getopts or somethintg

Reply:
mm good point